spec/lib/hello_trema_spec.rb

```diff
@@ -2,14 +2,19 @@
 require 'hello_trema'
 
 describe HelloTrema do
-  Given(:hello_trema) { HelloTrema.new }
+  Given(:logger) { spy('logger') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.